libs/tools/barista/src/builder/strapi.ts

```diff
@@ -96,9 +117,28 @@ export const strapiBuilder: BaPageBuilder = async (
     transformed.push({ pageContent, relativeOutFile });
   }
 
+  categories.push(...categoriesData.map((category) => category.title));
+
+  const distDir = next ? 'dist/next-data' : 'dist/barista-data';
```

Comment:
Can you make this configurable via the options that are passed to the barista builder – there should be an option for the `outputPath` that points to the next data instead of the barista data for the other build no hardcoded paths inside the code makes refactoring hard.